conformance_tests/core/test_cmdqueue/src/test_cmdqueue.cpp

```diff
@@ -235,6 +235,155 @@ TEST_P(
   }
 }
 
+class zeImmediateCommandListAppendCommandListsExpTests
+    : public ::testing::Test,
+      public ::testing::WithParamInterface<CustomExecuteParams> {
+protected:
+  void SetUp() override {
+    ze_device_handle_t device = lzt::zeDevice::get_instance()->get_device();
+    const ze_driver_handle_t driver = lzt::get_default_driver();
+    const ze_context_handle_t context = lzt::get_default_context();
+    EXPECT_GT(params.num_command_lists, 0);
+
+    print_cmdqueue_exec(params.num_command_lists, params.sync_timeout);
+
+    ze_command_queue_desc_t cmdQueueDesc = {
```

Comment:
We typically stick to snake_case in the CTS, not camelCase

Reply:
I am trying to use snake_case but looking at the files it appears that the styles are all over the place.

level-zero-tests/utils/test_harness/src/test_harness_cmdlist.cpp